engine-nextgen/src/test/kotlin/graphql/nadel/test/Debug.kt

```diff
@@ -0,0 +1,22 @@
+package graphql.nadel.test
+
+/**
+ * Util function to debug e.g.
+ *
+ * ```kotlin
+ * fun main() {
+ *   println(dbg(1 + 2 + 3))
+ * }
+ * ```
+ *
+ * prints the following
+ *
+ * ```
+ * dbg(1 + 2 + 3)
+ *       |   |
+ *       |   6
+ *       3
+ * 6
+ * ```
+ */
+fun <T> dbg(value: T): T = value
```

Comment:
how out of interest??

Reply:
It's using https://github.com/bnorm/kotlin-power-assert throughout this module

So any `assert`s are the Groovy like ones 😄